src/Builder/ClassModifier/ClassInstanceClassModifierInterface.php

```diff
@@ -10,38 +10,33 @@
 namespace SprykerSdk\Integrator\Builder\ClassModifier;
 
 use SprykerSdk\Integrator\Transfer\ClassInformationTransfer;
+use SprykerSdk\Integrator\Transfer\ClassMetadataTransfer;
 
 interface ClassInstanceClassModifierInterface
 {
     /**
      * @param \SprykerSdk\Integrator\Transfer\ClassInformationTransfer $classInformationTransfer
      * @param string $targetMethodName
-     * @param string $classNameToAdd
-     * @param string $before
-     * @param string $after
-     * @param string|null $index
+     * @param \SprykerSdk\Integrator\Transfer\ClassMetadataTransfer $classMetadataTransfer
      *
      * @return \SprykerSdk\Integrator\Transfer\ClassInformationTransfer
      */
     public function wireClassInstance(
         ClassInformationTransfer $classInformationTransfer,
         string $targetMethodName,
-        string $classNameToAdd,
-        string $before = '',
-        string $after = '',
-        ?string $index = null
+        ClassMetadataTransfer $classMetadataTransfer
     ): ClassInformationTransfer;
 
     /**
      * @param \SprykerSdk\Integrator\Transfer\ClassInformationTransfer $classInformationTransfer
-     * @param string $classNameToRemove
      * @param string $targetMethodName
+     * @param \SprykerSdk\Integrator\Transfer\ClassMetadataTransfer $classMetadataTransfer
      *
      * @return \SprykerSdk\Integrator\Transfer\ClassInformationTransfer|null
      */
     public function unwireClassInstance(
         ClassInformationTransfer $classInformationTransfer,
-        string $classNameToRemove,
-        string $targetMethodName
+        string $targetMethodName,
+        ClassMetadataTransfer $classMetadataTransfer
```

Comment:
Should `$targetMethodName` property should be part of `ClassMetadataTransfer`?

Reply:
Thanks! Moved `targetMethodName` to `ClassMetadataTransfer`